src/js/dropdown-menu/dropdown.underscore

```diff
@@ -1,5 +1,11 @@
 <a href="<%- main.url %>" class="user-title">
-    <span class="sr"><%- main.screenreader_label %> </span><%- main.text %>
+    <% if (main.screenreader_label) { %>
+    	<span class="sr"><%- main.screenreader_label %> </span>
+	<% } %>
+	<% if (main.user_image) { %>
```

Comment:
I wouldn't call this "user_image" as this is intended to be a generic dropdown, not a user dropdown. We happen to show users in the platform, but I could imagine drop downs with other kinds of images.

Hmmm, I see that the overall class is "user_title". I'm thinking we should change this too.